src/pages/monitor/Monitor.tsx

```diff
@@ -66,7 +76,12 @@ const Endpoint: React.FC<Endpoint> = ({ url, locationEndPoint, disable }) => {
   ]
 
   return (
-    <div className={disable ? 'endpoint disable' : 'endpoint'}>
+    <div
+      className={
+        disable ? 'endpoint disable cursor-pointer' : 'endpoint cursor-pointer'
+      }
+      onClick={handleClickEndpoint}
+    >
       <div className="endpoint-flag-container">
```

Comment:
Should we add a "copy to clipboard" icon somewhere in the UI and use that to handle the copy instead of the entire row? something like https://www.flaticon.com/free-icon/copy_126498

Reply:
Hi Max!
The action "copy to clipboard" is only available for the endpoint column, not the entire row. We can add a button, but where exactly should we add it? Before country flags or after endpoint address?
Since the endpoint addresses are variable in length, and all flags are aligned, maybe the first option would be more consistent? What do you think?

Reply:
Hi @comountainclimber, to further contextualize around the rationale for what was done here: due to the reasons described by Kenedy before, we understood that it would probably require some design adjustments to include that button in a way that it doesn't impact the UI, as everything would be cramped if we added it before or after the addresses. So, we thought that it would be sufficient for a first release to allow users to click on the end point to copy it, while also providing a tool tip like "click here to copy the address" when hovering over the endpoint.
We would use the same tooltip element that @endkeyCoder is working on in #310, using the same UI shown in the design.
Please let us know if that's reasonable for the first release or if we should involve James to help us including an icon somewhere in the table (possibly adjusting it a bit) in a way that is aesthetically pleasing.

Reply:
Hi @comountainclimber, please ignore my last comment as it seems we can add it to the left of the endpoint address without messing with the UI 😄

Reply:
No worries! For now let’s go and do that I think it will be more clear to users this way